plutus-core/untyped-plutus-core/test/Transform/callsiteInline.uplc.golden

```diff
@@ -2,7 +2,7 @@
   (lam
     a_5
     [
-      [ f_1 [ [ a_5 (con integer 0) ] (con integer 1) ] ]
+      [ f_1 [ [ g_2 (con integer 1) ] (con integer 0) ] ]
```

Comment:
okay, now _this_ looks busted. That's a free variable where we didn't have one before.

Reply:
Oh wait, `g` is free in the input! and a is bound to g. So this is actually fine, we've just inlined a, which seems okay

Reply:
I'm not sure I understand. These two outputs are semantically different, I think. Also I don't know why we say *inline* `a` here, I think this is beta reduction?

Reply:
Sorry, a is not bound to g. a is bound to `\x y -> g y x`. This is UPLC, so it's an immediately applied lambda, see `a` at line 3 and the definition on line 9. We've indeed _inlined_ one occurrence of `a`, replacing it with `\x y -> g y x`, and then reducing that lambda. We haven't beta reduced the original application - it's still there!

Reply:
I see that now. Thanks!

Reply:
I guess we just didn't notice that `a` wasn't inlined before! Great news that this is more right now.